articles/modules/ROOT/pages/protecting-against-SSRF.adoc

```diff
@@ -0,0 +1,52 @@
+= Protecting against Server Side Request Forgery (SSRF)
+:slug: protecting-against-SSRF
+:author: Irene Michlin
+:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4, 5.0
+:tags: cypher, security
+:category: security
```

Comment:
4.0 is not being covered by the back-port. The reason being that it's 1 year old, and that we don't have any customers with contracts requiring active support for that version. 

For context (don't add to this article) we may include 3.4 in the future, depending on whether it is a requirement for the only 3.4 customer we have there (Nokia).

Reply:
That was copypaste from another article, let's delete 4.0